frontend/src/components/public_group/AmountPicker.js

```diff
@@ -44,7 +44,7 @@ export default class AmountPicker extends React.Component {
         )}
 
         <div className='max-width-1 mx-auto' style={{position: 'absolute', left: '0', right: '0', margin: '-25px auto'}}>
-          {!hasStripe && (
+          {(!hasStripe || (hasPaypal && hasStripe)) && (
```

Comment:
No, not if both hasStripe and hasPaypal are true, I believe this is the case during the tests, the reason the button is missing.